### DISCUSSION TITLE
NAS 如何安装

### DISCUSSION THREAD

**Discussion by ghost** [UIforFreedom/UIF#27] [General]
我的 NAS 是飞牛 NAS，请问一下如何安装。

**Comment by UIforFreedom** [comment]
用 docker 安装

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

2024 年 10 月 12 日星期六 12:27, 西裝狂徒 ***@***.***(mailto:2024 年 10 月 12 日星期六 12:27, 西裝狂徒 <<a href=)> 来信：

**Comment by ghost** [comment]
我已经安装成功。这个工具有订阅自动更新功能吗。在 NAS 上，基本后期就很少打开他。所有有这个功能就不用手动打开更新订阅了。
还有就是分流规则可以导入分流文件吗？

UIforFreedom ***@***.***> 于 2024年10月12日周六 下午3:06写道：

**Comment by UIforFreedom** [comment]
你所说的功能都不支持，也没支持的打算

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

2024 年 10 月 13 日星期日 05:23, 西裝狂徒 ***@***.***(mailto:2024 年 10 月 13 日星期日 05:23, 西裝狂徒 <<a href=)> 来信：
